internal/pkg/metadb/metadb.go

```
@@ -524,3 +527,49 @@ func (m *MetaDB) ListBlobRefsByStatus(ctx context.Context, status blobref.Status
 	iter := blobref.NewCursor(m.client.Run(ctx, query))
 	return iter, nil
 }
+
+func addPropertyFilter(q *ds.Query, f *pb.QueryFilter) *ds.Query {
+	switch f.Operator {
+	case pb.FilterOperator_EQUAL:
+		return q.Filter(fmt.Sprintf("Properties.%s =", f.PropertyName), record.ExtractValue(f.Value))
+	default:
+		// TODO(hongalex): implement inequality filters
```

Return Unimplemented error here instead of silently ignoring?